src/bundles/SellerRegistration/components/RecruiterForm/RecruiterForm.js

```diff
@@ -263,7 +263,7 @@ class RecruiterForm extends BaseForm {
                                         id="LabourHire"
                                         messages={this.generateLicenceMessages()}
                                     />
-                                    <p>Some states operate under a mandated Labour Hire Licensing Act. You will require a relevant licence if you are applying for specialist opportunities in VIC, SA or QLD.</p>
+                                    <p>Some states operate under a mandated Labour Hire Licensing Act. You will require a relevant licence if you are applying for specialist opportunities in VIC or QLD.</p>
```

Comment:
I don't know why these were abbreviated but I think we should use the names of the states.
```suggestion
                                    <p>Some states operate under a mandated Labour Hire Licensing Act. You will require a relevant licence if you are applying for specialist opportunities in Victoria or Queensland.</p>
```